WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Any reason for this? I'm actually surprised rubocop is okay with this blank line on top of the file.